src/qt/clientmodel.cpp

```diff
@@ -160,18 +160,8 @@ void ClientModel::updateNumConnections(int numConnections)
     Q_EMIT numConnectionsChanged(numConnections);
 }
 
-void ClientModel::updateAlert(const QString& hash, int status)
-{
-    // Show error message notification for new alert
-    if (status == CT_NEW) {
-        uint256 hash_256;
-        hash_256.SetHex(hash.toStdString());
-        CAlert alert = CAlert::getAlertByHash(hash_256);
-        if (!alert.IsNull()) {
-            Q_EMIT message(tr("Network Alert"), QString::fromStdString(alert.strStatusBar), CClientUIInterface::ICON_ERROR);
-        }
-    }
-
+void ClientModel::updateAlert()
+{
     Q_EMIT alertsChanged(getStatusBarWarnings());
```

Comment:
`alertChanged` signal is not connected to the GUI at all, can remove it entirely from `clientModel`. Including the `NotifyAlertChanged` boost event binding.

Reply:
yes this is something we've discussed before. alert type messages need to be presented in the GUI in a future PR.